src/main/java/core/basesyntax/dataparser/DataParserStrategyImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.dataparser;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DataParserStrategyImpl implements DataParserStrategy {
```

Comment:
remove this strategy. Your terms of reference do not specify any other possible formats. Do not introduce your own requirements. Also, you didn't provide any implementations other than csv.